java/runtime/src/main/java/io/ray/runtime/RayNativeRuntime.java

```diff
@@ -54,7 +54,7 @@ public RayNativeRuntime(RayConfig rayConfig) {
 
   private void updateSessionDir(GcsClient gcsClient) {
     // Fetch session dir from GCS.
-    final String sessionDir = gcsClient.getInternalKV("session_dir");
+    final String sessionDir = gcsClient.getInternalKV("@:session:session_dir");
```

Comment:
How about adding namespace arg for this client too?

Reply:
My concern is about the compatibility actually. Hmm, but maybe it's ok? Maybe after fixing major issues in gcs ha, but before 1.10?